### PR TITLE
DOCS-Update-supported-versions

### PR DESCRIPTION
**Pull request by bmckercher123**
3.17 Support and compability doc

Product Version(s):
CE 3.17.0

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx [hunk 1]
```diff
@@ -37,79 +37,46 @@ Note that all Windows feature limitations are described in [Windows limitations]
...
-| 3.7                  | - GKE CNI with {{prodname}} network policy |
-| 3.6                  | - GKE CNI with {{prodname}} network policy |
-| 3.5                  | - GKE CNI with {{prodname}} network policy |
+| 3.5 through current release                  | - GKE CNI with {{prodname}} network policy |
 
 ## kOps on AWS
 
 | {{prodname}} version | kOps and Kubernetes versions | {{prodname}} support                                         |
 | -------------------- | ---------------------------- | ------------------------------------------------------------ |
+| 3.17                 | 1.24 - 1.25                  | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx:65]:
```suggestion
| 3.17                 | 1.25 - 1.26                  | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
```

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx [hunk 2]
```diff
@@ -37,79 +37,46 @@ Note that all Windows feature limitations are described in [Windows limitations]
...
-| 3.8                  | 1.20                         | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
-| 3.7                  | 1.20                         | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
-| 3.6                  | 1.19                         | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
+| 3.6 - 3.9            | 1.20                         | - {{prodname}} CNI with network policy<br />- AWS CNI with{{prodname}} network policy |
 
 ## Kubernetes-kubeadm
 
 | {{prodname}} version | Kubernetes/kubeadm versions | {{prodname}} support                 |
 | -------------------- | --------------------------- | ------------------------------------ |
+| 3.17                 | 1.24 - 1.26                 | {{prodname}} CNI with network policy |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx:79]:
```suggestion
| 3.17                 | 1.25 - 1.27                 | {{prodname}} CNI with network policy |
```

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx [hunk 3]
```diff
@@ -144,6 +112,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | OpenShift versions for Kubernetes | {{prodname}} support                 |
 | -------------------- | --------------------------------- | ------------------------------------ |
+| 3.17                 | 4.10 - 4.12                        | {{prodname}} CNI with network policy |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx:115]:
```suggestion
| 3.17                 | 4.12 - 4.13                        | {{prodname}} CNI with network policy |
```

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx [hunk 4]
```diff
@@ -178,6 +148,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | {{prodname}} support                 | Kubernetes versions |
 | -------------------- | ------------------------------------ | ------------------- |
+| 3.17                 | {{prodname}} CNI with network policy | 1.24 - 1.25         |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx:151]:
```suggestion
| 3.17                 | {{prodname}} CNI with network policy | 1.25 - 1.26         |
```

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx [hunk 5]
```diff
@@ -161,6 +130,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | RKE version | {{prodname}} support                 | Kubernetes versions |
 | -------------------- | ----------- | ------------------------------------ | ------------------- |
+| 3.17                 | 1.4         | {{prodname}} CNI with network policy | 1.23                |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx:133]:
```suggestion
| 3.17                 | 1.4         | {{prodname}} CNI with network policy | 1.25                |
```

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx (hunk 3 above)
Comment (sabags) [reply calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx:115]: Should this be 4.11 - 4.13 ?

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx:65]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx:79]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx (hunk 3 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx:115]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx (hunk 5 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx:133]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx (hunk 4 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx:151]: Fixed.

**Pull request closed without merging**